targets/TARGET_STM/can_api.c

```diff
@@ -491,6 +573,17 @@ void FDCAN2_IT1_IRQHandler(void)
     can_irq(CAN_2, 1);
 }
 
+void FDCAN3_IT0_IRQHandler(void)
```

Comment:
#if defined(FDCAN3_BASE) is missing

Reply:
I didn't add this line, since it was also missing for the FDCAN2 Handlers, which otherwise are also guarded with the #if defined(GDCAN2_BASE). 

But I agree, it makes sense to add the guard for both the FDCAN2 and FDCAN3 Handlers.

Reply:
Added guards in 35c9e7a